FEXCore/Source/Interface/Core/ArchHelpers/CodeEmitter/ALUOps.inl

```diff
@@ -766,6 +766,11 @@ public:
     EvaluateIntoFlags(Op, 1, rn);
   }
 
+  void cfinv() {
```

Comment:
This needs a unittest in `FEXCore/unittests/Emitter/ALU_Tests.cpp` probably near the setf16 test.